colossalai/inference/core/config.py

```diff
@@ -41,14 +42,14 @@ class InferenceConfig:
     max_seq_len: Optional[int] = None
     quant_mode: Optional[str] = None
     revision: Optional[str] = None
+    # TODO: beam search is not support for now
+    beam_width: int = 1
```

Comment:
Shouldn't this be one of the argument of `generate`?

Reply:
Ok, then you should add its description to the docstring as well.

Reply:
Done